docs/troubleshooting/faqs/TypeScript.mdx

```diff
@@ -52,3 +52,27 @@ You can use ESLint to enforce good uses of both ESLint and TypeScript comment di
 - The [`@eslint-community/eslint-plugin-eslint-comments`](https://eslint-community.github.io/eslint-plugin-eslint-comments) plugin can enforce general ESLint comment best practices, including requiring descriptions
 
 :::
+
+## How should I handle reports that conflict with [`verbatimModuleSyntax`](https://www.typescriptlang.org/tsconfig/#verbatimModuleSyntax)?
+
+Several TypeScript options impact how imports and exports are handled in your project, including
+
+- [`verbatimModuleSyntax`](https://www.typescriptlang.org/tsconfig/#verbatimModuleSyntax)
+- [`importsNotUsedAsValues`](https://www.typescriptlang.org/tsconfig/#importsNotUsedAsValues)
+- [`preserveValueImports`](https://www.typescriptlang.org/tsconfig/#preserveValueImports)
+- [`esModuleInterop`](https://www.typescriptlang.org/tsconfig/#esModuleInterop)
+- [`allowSyntheticDefaultImports`](https://www.typescriptlang.org/tsconfig/#allowSyntheticDefaultImports)
+
+Additionally, whether one is authoring ES Modules or CommonJs impacts `import`/`export`/`require` semantics.
+Therefore, some of our rules may not apply at all or may need special configuration in projects using these options.
```

Comment:
* Unnecessary conjunctive adverb - it's implied by the two sentences being together
* Unnecessary "at all" - it doesn't change the sentence's meaning

```suggestion
Some of our rules may not apply or may need special configuration in projects using these options.
```